Makefile

```diff
@@ -50,6 +50,9 @@ ontologies/vbo-edit.owl:
 ontologies/chr.owl: 
 	$(ROBOT) convert -I https://raw.githubusercontent.com/monarch-initiative/monochrom/master/chr.owl -o $@.tmp.owl && mv $@.tmp.owl $@
 
+ontologies/omim.owl: 
+	$(ROBOT) convert -I https://github.com/monarch-initiative/omim/releases/latest/download/omim.owl -o $@.tmp.owl && mv $@.tmp.owl $@
```

Comment:
The page for this URL says `do not use omim.owl, it has been attached as a test only`. Does that mean that as a "Release" this URL will be stable, but obviously static for the version of OMIM in that release? 

Also, what are you plans for future omim releases like this Test _or_ is our tracking about this, ie this is the source of the OMIM file, just this conversation? The description is good and makes the source very clear!

Reply:
> Does that mean that as a "Release" this URL will be stable, but obviously static for the version of OMIM in that release?

There is a  PR open on the OMIM repo (https://github.com/monarch-initiative/omim/pull/107) which will add this omim.owl file to every new release, which is automatically updated every week by github actions. I just uploaded it there by hand now so that this OLS PR does not have to wait until Joe has checked the PR there..

Future plans.. Good question. Maybe discuss in meeting?